packages/data/storage.ts

```diff
@@ -15,3 +15,39 @@ export const Localstorage = {
   LocaleStore: 'locale.store',
   AlgorithmStore: 'algorithm.store'
 };
+
+// Cookies keys
+
+export const CookiesKeys = {
+  AccessToken: 'accessToken',
+  RefreshToken: 'refreshToken'
+};
+
+type Keys = keyof typeof CookiesKeys;
+export type CookiesKey = (typeof CookiesKeys)[Keys];
+
+class CookieStorage {
```

Comment:
if we don't use helper we have to install js-cookie package everywhere we want to use cookie

Reply:
Yes, especially if take in mind, this code from my pr.

Reply:
> we dont want this helper, we can use Cookies.something directly in web app, it should not live inside data folder

developers always takes inspiration from others ok

Reply:
> we dont want this helper, we can use Cookies.something directly in web app, it should not live inside data folder

i used helper so that we can reduce dependency installs, if don't use helper "js-cookie"
dependency also need to be installed in lens package

Reply:
That's fine, data package is not meant to store utils, each should have their own and it should run even it's ejected from monorepo

Reply:
removed helper and renamed CookiesKey